package.json

```
@@ -44,7 +44,7 @@
     "archy": "^1.0.0",
     "bluebird": "^3.7.2",
     "hexo-cli": "^4.3.0",
-    "hexo-front-matter": "4.0.0",
+    "hexo-front-matter": "^4.0.0",
```

Just now I published `v4.2.0`.

https://www.npmjs.com/package/hexo-front-matter/v/4.2.0

```suggestion
    "hexo-front-matter": "^4.2.0",
```

---

Also, we should change the PR title.